src/css/styles.scss

```diff
@@ -0,0 +1,170 @@
+@import './globalStyles';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Open Sans', sans-serif;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background-color: $primary-background;
+}
+
+main {
+  @include flex-center();
+
+  height: 100vh;
+}
+
+#to-do-container {
+  width: 25%;
```

Comment:
I will suggest that you give the width of this #to-do-container in pixels, say 500px, and add a max-width of 90% also so that for smaller viewports such as the mobile screens, your todo list can be visible.